From fdc867288697d8b052145e80911d2d338d7d02b7 Mon Sep 17 00:00:00 2001
From: Anton Khirnov <anton@khirnov.net>
Date: Wed, 31 Oct 2012 22:10:41 +0100
Subject: [PATCH 154/278] audiointerleave: deobfuscate a function call.

right above there is if (pkt) {<do stuff>; pkt = NULL}, so pkt is just a
fancy name for NULL at this point.
---
 libavformat/audiointerleave.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libavformat/audiointerleave.c b/libavformat/audiointerleave.c
index e48f826..5df0bb0 100644
--- a/libavformat/audiointerleave.c
+++ b/libavformat/audiointerleave.c
@@ -130,5 +130,5 @@ int ff_audio_rechunk_interleave(AVFormatContext *s, AVPacket *out, AVPacket *pkt
         }
     }
 
-    return get_packet(s, out, pkt, flush);
+    return get_packet(s, out, NULL, flush);
 }
-- 
1.7.9.5

